### PR TITLE
Fix logits

### PR DESCRIPTION
**Pull request by daniv-20**

### CODE REVIEW

**Comment by daniv-20** [comment]
Added support for handling binary variables. Still need to add tests for binary vars but will fix R first.
